bindings/python/opendaq/generated/reader/py_stream_reader.cpp

```diff
@@ -25,73 +14,255 @@
  * limitations under the License.
  */
 
-#include "py_opendaq/py_opendaq.h"
 #include "py_core_types/py_converter.h"
+#include "py_opendaq/py_opendaq.h"
 
-#include <pybind11/stl.h>
 #include <pybind11/chrono.h>
+#include <pybind11/stl.h>
+
+using SampleTypeVariant = std::variant<py::array_t<daq::SampleTypeToType<daq::SampleType::Float32>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::Float64>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::UInt32>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::Int32>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::UInt64>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::Int64>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::UInt8>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::Int8>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::UInt16>::Type>,
+                                       py::array_t<daq::SampleTypeToType<daq::SampleType::Int16>::Type>>;
```

Comment:
This will likely be used in other readers as well, right? In that case, it might make more sense to add it to a header file that gets included by all readers.

Reply:
This will highly likely be used in other readers, so it could be moved in the future updates